src/components/Buttons/Button/Button.vue

```diff
@@ -129,11 +129,15 @@ export default {
         ? `var(--color-${this.color})`
         : 'var(--color-primary)';
 
+      const color = getComputedStyle(document.documentElement).getPropertyValue(`--color-${this.color || 'primary'}`);
```

Comment:
the default color is already primary, why are u using this `|| 'primary'`?

Reply:
removed the || primary